charts/keycloakx/templates/ingress.yaml

```diff
@@ -0,0 +1,131 @@
+{{- $ingress := .Values.ingress -}}
+{{- if $ingress.enabled -}}
+{{- $apiV1 := false -}}
+{{- $apiVersion := "extensions/v1beta1" -}}
+{{- if and (.Capabilities.APIVersions.Has "networking.k8s.io/v1") (semverCompare ">= v1.19.0-0" .Capabilities.KubeVersion.Version) -}}
+  {{- $apiVersion = "networking.k8s.io/v1" -}}
+  {{- $apiV1 = true -}}
+{{- else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" -}}
+  {{- $apiVersion = "networking.k8s.io/v1beta1" -}}
+{{- else -}}
+  {{- $apiVersion = "extensions/v1beta1" -}}
```

Comment:
Do we plan to drop support for those old api version at some point in time? This might simplify the ingress resource templates.

Reply:
Good point, which api versions should we keep?

Reply:
v1 was stable with Kubernetes v1.19.0, which was released on Aug 26, 2020. So I guess that should have been enough time to upgrade. I would suggest to keep v1 and maybe add other versions again if someone requests it via an issue.

Reply:
Thanks again! I just removed support for the version switch.